swri_transform_util/include/swri_transform_util/local_xy_util.h

```diff
@@ -80,14 +80,25 @@ namespace swri_transform_util
       double& longitude);
 
   /**
-   * Utility class for converting between WGS84 lat/lon and an ortho-rectified
+   * @brief Utility class for converting between WGS84 lat/lon and an ortho-rectified
```

Comment:
`@brief` is necessary unless `autobrief` is turned on, and in `rosdoc` it isn't. You can see [here](http://docs.ros.org/indigo/api/swri_transform_util/html/classswri__transform__util_1_1UtmUtil.html) that none of the methods of `UtmUtil` have briefs, even though the [source code](https://github.com/swri-robotics/marti_common/blob/master/swri_transform_util/include/swri_transform_util/utm_util.h) has documentation.